docs/product-installation-guide/src/main/asciidoc/controller-jws-install-proc.adoc

```diff
@@ -0,0 +1,50 @@
+[id='controller-standalone-jws-install-proc']
+= Installing the Controller on Red Hat JBoss Web Server
+
+You can install the Controller and use the REST API or the {KIE_SERVER} Java Client API to interact with it.
+
+.Prerequisites
+* A backed up Red Hat JBoss Web Server 3.1 with Tomcat 8 or higher server installation. The base directory of the JBoss Web Server installation is referred to as `__JWS_HOME__`. 
+* Sufficient user permissions to complete the installation.
+* The `rhdm-7.0-controller-jws.zip` file. To download this file:
++
+--
+.. Click *Download* next to *{PRODUCT} 7.0.0 Add Ons* on the Product Downloads page of the https://access.redhat.com[Red Hat Customer Portal].
+.. Unzip the `rhdm-7.0.0.GA-add-ons.zip` file. The `rhdm-7.0-controller-jws.zip` file is in the unzipped directory.
+--
+
+.Procedure
+. Extract the `rhdm-7.0-controller-jws.zip` archive to a temporary directory. In the following examples, this directory is called `__TEMP_DIR__`.
+. Copy the `__TEMP_DIR__/rhdm-7.0-controller-jws.zip/controller.war` directory to the `_JWS_HOME_/tomcat8/webapps` directory.
++
+WARNING: Ensure the names of the {PRODUCT} deployments you are copying do not conflict with your existing deployments in the Red Hat JBoss Web Server instance.
+. Remove the `.war` extensions from the `controller.war` folder.
+. Copy the contents of the `__TEMP_DIR__/rhdm-7.0-controller-jws/SecurityPolicy/` directory to `__JWS_HOME__/bin`. When asked to overwrite files, select *Yes*.
+. In the `__JWS_HOME__/standalone/deployments/` directory, create an empty file named `controller.war.dodeploy`. This file ensures that the Controller is automatically deployed when the server starts.
+. Add the `kie-server` role and user to the `_JWS_HOME_/tomcat8/conf/tomcat-users.xml` file. In the following example,  `<USERNAME>` and `<PASSWORD>` is a user and password combination of your choice:
++
+[source]
+----
+<role rolename="<kie-server>"
+<user username="<USER_NAME> password="<PASSWORD>" roles="<kie-server>"/>
```

Comment:
Should be
`
<user username="<USER_NAME>" password="<PASSWORD>" roles="kie-server"/>
`